src/sentry/api/helpers/group_index/update.py

```diff
@@ -300,6 +299,34 @@ def update_groups(
             res_type = GroupResolution.Type.in_next_release
             res_type_str = "in_next_release"
             res_status = GroupResolution.Status.pending
+        elif status_details.get("inUpcomingRelease"):
+            if len(projects) > 1:
+                return Response(
+                    {"detail": "Cannot set resolved in next release for multiple projects."},
```

Comment:
shoudl this response detail text be changed to upcoming release?

Reply:
good catch , will update